keadm/app/cmd/util/common.go

```diff
@@ -38,9 +38,10 @@ const (
 	DefaultDownloadURL = "https://download.docker.com"
 	DockerPreqReqList  = "apt-transport-https ca-certificates curl gnupg-agent software-properties-common"
 
-	KubernetesDownloadURL = "https://apt.kubernetes.io/"
-	KubernetesGPGURL      = "https://packages.cloud.google.com/apt/doc/apt-key.gpg"
-
+	KubernetesDownloadURL     = "https://apt.kubernetes.io/"
+	KubernetesGPGURL          = "https://packages.cloud.google.com/apt/doc/apt-key.gpg"
```

Comment:
Rename this as KubernetesGPGURL - >KubernetesUbuntuGPGURL , Modify it where ever it is used.